.vscode/c_cpp_properties.json

```
@@ -0,0 +1,24 @@
+{
+    "configurations": [
+        {
+            "name": "Linux-GCC",
+            "includePath": [
+                "${workspaceFolder}/**",
+                "/usr/local/cuda/include/**" // required to find the cuda runtime
```

I think that is not valid json?
```suggestion
                "/usr/local/cuda/include/**"
```